doc/other-languages.md

```diff
@@ -32,7 +32,8 @@ sourmash_comp_matrix <- as.matrix(sourmash_comp_matrix)
 
 ```
 
-[See the output of plotting and clustering this matrix](_static/ecoli-cmp.html), produced by [this RMarkdown file](_static/ecoli-cmp.Rmd).
+[See the output of plotting and clustering this matrix](https://sourmash.readthedocs.io/en/latest/_static/ecoli-cmp.html),
+produced by [this RMarkdown file](https://sourmash.readthedocs.io/en/latest/_static/ecoli-cmp.Rmd).
 
-You can download the `ecoli.cmp.csv` file itself [here](_static/ecoli.cmp.csv).
+You can download the `ecoli.cmp.csv` file itself [here](https://sourmash.readthedocs.io/en/latest/_static/ecoli.cmp.csv).
```

Comment:
same, local static CSV files aren't reference-able? If not, we might consider pointing directly to the GitHub URL since they have a decent CSV renderer.

Reply:
punted to https://github.com/dib-lab/sourmash/issues/1051